src/components/menu-bar/author-info.css

```diff
@@ -0,0 +1,51 @@
+@import "../../css/colors.css";
+@import "../../css/units.css";
+
+
+
+.author-info {
+    color: $ui-white;
+    font-family: "Helvetica Neue";
+    display: flex;
+    justify-content: start;
+    align-items: center;
+}
+
+.author-info .avatar {
+    margin-right: .5625rem;
+}
+
+.author-info .title-author {
+    /* display: flex;
+    align-items: center;
+    justify-content: end; */
+}
+
+.author-info .project-title {
```

Comment:
The beauty of css modules is that it creates localized class names, so you don't have to do this prefixing for specificity's sake. In fact putting this prefixing here makes it more brittle since it couples the style to the element hierarchy unnecessarily. Nothing about a `.project-title` being a child of `.author-info` matters for the font-size, line-height, etc.

Reply:
That makes sense